src/components/home/Home.js

```diff
@@ -17,23 +17,18 @@ export default function Home({ token }) {
       const link = pet.primary_photo_cropped;
       return (
         <Col md={4} xs={12} key={pet.id}>
-          <Card style={{ margin: 10, paddingTop: 10, width: "100%" }}>
+          <Card className = "card">
             <Card.Img
               variant="top"
               alt={link ? pet.type : `${pet.type} placeholder`}
               src={link ? link.medium : placeholder}
             />
-            <Card.Body
-              style={{
-                flex: 1,
-                alignSelf: "center",
-              }}
-            >
+            <Card.Body className="cardBody">
               <Card.Title style={{ textAlign: "center" }}>
```

Comment:
This needs to be in the css. Since we are using bootstrap's css we need to specify that we waint it to be changed in our home page. Our parent div `home__container` and the name of `Card.Title` for the css would be `.card-title`. To set it up to be only changed in home we can do 
```
.home__container .card-title{ 
//css here
}
```

in the `home.css`